docs/usage.md

```diff
@@ -0,0 +1,102 @@
+---
+parent: VERTEX-CFD v1.0 User Guide
+title: Usage
+nav_order: 3
+usemathjax: true
+---
+
+# Usage
+
+Once installed, VERTEX-CFD relies on two files. The first one is the 'vertexcfd' executable and the second one is the input file for the simulation. After the installation, there are two executables, the first one is located in `BUILD_PATH/src/vertexcfd` and the second one is located in `INSTALL_PATH/bin/vertexcfd`. Both of the executables should work. The input file is case spesific and there are example case files in `vertex-cfd/examples/inputs`. In this document, the input file located in `vertex-cfd/examples/inputs/incompressible/incompressible_2d_channel.xml` will be used as an example case. 
```

Comment:
I think we should try to only user input files that are located in `INSTALL_PATH`.